detection_rules/etc/api_schemas/master/master.base.json

```diff
@@ -364,36 +367,10 @@
     },
     "timeline_id": {
       "description": "TimelineTemplateId",
-      "enum": [
-        "db366523-f1c6-4c1f-8731-6ce5ed9e5717",
-        "91832785-286d-4ebe-b884-1a208d111a70",
-        "76e52245-7519-4251-91ab-262fb1a1728c",
-        "495ad7a7-316e-4544-8a0f-9c098daee76e",
-        "4d4c0b59-ea83-483f-b8c1-8c360ee53c5c",
-        "e70679c2-6cde-4510-9764-4823df18f7db",
-        "300afc76-072d-4261-864d-4149714bf3f1",
-        "3e47ef71-ebfc-4520-975c-cb27fc090799",
-        "3e827bab-838a-469f-bd1e-5e19a2bff2fd",
-        "4434b91a-94ca-4a89-83cb-a37cdc0532b7"
-      ],
-      "enumNames": [],
       "type": "string"
     },
     "timeline_title": {
       "description": "TimelineTemplateTitle",
-      "enum": [
-        "Generic Endpoint Timeline",
-        "Generic Network Timeline",
-        "Generic Process Timeline",
```

Comment:
Why is this being removed?

Reply:
Invariantly when following steps i performed 
```
python -m detection_rules dev schemas update-rule-data
```

These changes was part of those? Should we revert it and just keep Monthly refresh to ECS Beats and Integrations

Reply:
I do not recall specifically. Either this was originally removed (we should check historical PRs where we have refreshed) or it may have been introduced from DaC changes?

Reply:
Even in our Prep Branch we have had these similar changes in API_schema checks 
https://github.com/elastic/detection-rules/pull/3919/files#diff-9b73191af9f09924a03fb91927a045437901f36cba22c4887bf9912af9c87bc7